docs/docs/how-to/use-s3-presigned-url-to-upload-docs.md

```diff
@@ -4,195 +4,137 @@ title: Use S3 Signed URL to Upload Documents
 ---
 <div style={{paddingBottom:'24px'}}>
 
-In this how-to guide, you'll learn to upload documents to S3 buckets using the **S3 signed URL** from a ToolJet application. 
+In this how-to guide, we will upload documents to S3 buckets using the **S3 signed URL** from a ToolJet application.
 
-For this guide, We are going to use one of the existing templates on ToolJet: **S3 File explorer**
+For this guide, we are going to use one of the existing templates on ToolJet: **S3 File explorer**
+
+</div>
+
+<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+
+## Create an App Using Templates
 
 :::info using Templates
 On ToolJet Dashboard, Click on the down arrow on the right of the **New App** button, from the dropdown choose the **Choose from template** option. 
```

Comment:
This needs to be updated based on the latest UI. Also, this has to be a normal sentence, no need to add the info banner.

Reply:
The screenshot that has been added is from the latest UI. The template shown differs from the actual app we can access on Canvas.